windows.md

```diff
@@ -257,6 +257,9 @@ With those compatibility things out of the way, you're ready to start the system
     echo "export PGDATA=\"/c/Program Files/PostgreSQL/16/data\"" >> "$USERPROFILE/.bash_profile"
     echo "export PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> "$USERPROFILE/.bash_profile"
     source "$USERPROFILE/.bash_profile"
+    sed -i -e "s/^logging_collector = on/logging_collector = off/" \
+       -e "/^[[:space:]]*lc_messages[[:space:]]*=/ s/^#*.*$/lc_messages = 'en_US.UTF-8'/" \
+       "$PGDATA/postgresql.conf"
```

Comment:
This is running the two commands below. One is for setting the `logging_collector` to off, allowing `psql` to deliver logging info to the terminal

The second one sets the `lc_messages` to utf-8, allowing the messages in `psql` to be in English

```bash
sed -i "s/^logging_collector = on/logging_collector = off/" "$PGDATA/postgresql.conf"
sed -i "/^[[:space:]]*lc_messages[[:space:]]*=/ s/^#*.*$/lc_messages = 'en_US.UTF-8'/" "$PGDATA/postgresql.conf"
```

Reply:
let's keep them on separate lines - 2 `sed` commands

to make it easier to see what fails

Reply:
https://github.com/upleveled/system-setup/pull/80/commits/38accc6aa45bcea931e4bf60a869477723520245